cmd/kube-controller-manager/app/controllermanager.go

```diff
@@ -241,9 +241,14 @@ func (s *CMServer) Run(_ []string) error {
 	resourceQuotaController := resourcequotacontroller.NewResourceQuotaController(kubeClient)
 	resourceQuotaController.Run(s.ResourceQuotaSyncPeriod)
 
-	namespaceController := namespacecontroller.NewNamespaceController(kubeClient, s.NamespaceSyncPeriod)
+	namespaceController := namespacecontroller.NewNamespaceController(kubeClient, s.EnableHorizontalPodAutoscaler, s.NamespaceSyncPeriod)
 	namespaceController.Run()
```

Comment:
Maybe just a separate line so folks that want to put flags behind their feature its easier to see where they should add the code in the future.

```
// An OR of all flags to enable/disable experimental features
experimentalMode := s.EnableHorizontalPodAutoscaler
namespaceController := ....
```